client/src/utils/api.js

```diff
@@ -1,6 +1,5 @@
 // File to contain methods to return data from API calls
 import axios from "axios";
-import { BASE_URL } from "./constants";
```

Comment:
We removed this, but never added it back in, Where should base url be coming from if not an env?

Reply:
if we're importing env variables from dotenv in the top level of the app, then we can just import from that file right?

Reply:
If dotenv loads env vars in react apps, they're added to the `process.env` obj. so we would just reference `process.env.BASE_URL`

https://github.com/motdotla/dotenv

Docs have a really solid run down of this, but its also very similar to python's dotenv, js uses `process.env` with this 🖖🏻